woocommerce/woocommerce
# Try wrapping entire coming soon page under cover block to fix layout issue

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Another attempt to partially fix layout issues from https://github.com/woocommerce/team-ghidorah/issues/311 by wrapping everything under the cover block and flexbox positioning instead of using absolute positioning.

_Note that this does not yet fix the styling issues (such as changing background color, default margin/padding mismatches, etc.)_



### How to test the changes in this Pull Request:

1. Enable `launch-your-store` feature flag.
2. Go to `WooCommerce > Settings > Site Visibility` and enable `Coming soon` checkbox and save.
3. Go to `Appearance > Editor > Templates > Page: Coming soon` and edit it.
4. Try adding blocks around and see if the frontend layout matches.

https://github.com/woocommerce/woocommerce/assets/3747241/f9522cca-24d0-4829-b0d0-217b0a7e7302




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
